Build a tool bar(or header) and headline section

d4nQw3rty opened the pull request:
*Build a tool bar
*Build headline section

d4nQw3rty committed:
Update index.html

*Name title

d4nQw3rty committed:
Build a Tool Bar (or header)

*add button ternary image
*update index.html
*update stly.css

d4nQw3rty committed:
Update style.css

update style.css

d4nQw3rty committed:
build-head-line-section

*add primary text, primary content.
* add social media,
*add ul list tags

d4nQw3rty committed:
Update style.css

update style.css

d4nQw3rty committed:
Merge pull request #4 from d4nQw3rty/build-hedline-section

Build hedline section

d4nQw3rty committed:
Add images Icons

*Add images twitter, github, and other.
*update index.html
*uodate style.css

d4nQw3rty committed:
Merge pull request #5 from d4nQw3rty/add-images-to-media-icon-section

Add images Icons

d4nQw3rty committed:
Update index.html

*add medium and angellist  link

d4nQw3rty committed:
Update index.html

*add mudium, angellist link

danmainah commented on index.html:
```diff
@@ -5,9 +5,40 @@
-    <h1>hello</h1>
+    <div id="header-background-mobile">
+      <header id="Toolbar-Dark-Compact">
+         <div id="Logo-Placeholder">
+             <p id="My-Logo">Qwerty</p>
+         </div>
```
- [ ] Kindy ensure your logo is wrapped in an `a` tag, this is for the user experience as the ogo acts as a button to take the users to the homepage.

danmainah commented on index.html:
```diff
@@ -5,9 +5,40 @@
+         <div id="Logo-Placeholder">
+             <p id="My-Logo">Qwerty</p>
+         </div>
+         <div id="Normal-Button-Tertiary-Enabled">         
+            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
+         </div>
```
- [ ] Kindly ensure your hamburger menu is wrapped in a `nav` tag,  this tag is for semantics reasons and also helpful when positioning elements in mobile view.

danmainah commented on index.html:
```diff
@@ -5,9 +5,40 @@
+            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
+         </div>
+      </header>
+      <div id="Headline">
+         <h2 id="Primary-Text">I’m Danys
+             Glad to see you!
+         </h2>
+         <p id="Primary-Content">I’m a software developer! I can help you build
+             a product , feature or website Look through some
+             of my work and experience! If you like what you
+             see and have a project you need coded,
+             don’t hestiate to contact me.
+          </p>          
+          <div id="Social-Media">
+              <p id="Lets-Connect">LET’S CONNECT</p>            
+          </div>
+          <ul id="social-media-icons">
+            <li><a href="https://twitter.com/Danys_Linares" target="_blank" rel="noopener noreferer"><img src="img/Vector-twitter.svg" alt="Twitter link"></a></li>
+            <li><a href="https://www.linkedin.com/in/danys-linares-6a328b238?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BnkyI5IMjTzSg4PVJIZh%2BMw%3D%3D" rel="noopener noreferer"><img src="img/Linkedin-icon.svg" alt="Linkedin link"></a></li>
+            <li><a href="https://medium.com/" target="_blank" rel="noopener noreferer" ><img src="img/Vector-M.svg" alt="Medium link"></a></li>
+            <li><a href="https://github.com/d4nQw3rty" target="_blank" rel="noopener noreferrer"><img src="img/Vector-github.svg" alt="Github link"></a></li>
+            <li><a href="https://www.angellist.com/" target="_blank" rel="noopener noreferrer"><img src="img/Vector-hand.svg" alt="angellist link"></a></li>
+         </ul>                                  
+      </div>
+    </div>
```
- [ ] Kindly change your `div` tag to a `section` tag wrapping the whole section, the element represents a generic standalone section of a document, learn more[ here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

danmainah commented on index.html:
```diff
@@ -5,9 +5,40 @@
+         </div>
+      </header>
+      <div id="Headline">
+         <h2 id="Primary-Text">I’m Danys
+             Glad to see you!
+         </h2>
```
- [ ] Kindly change your header tag to h1 since this is your first header tag, this will make your page optimized for SEO.

danmainah reviewed (changes requested):
Hi @d4nQw3rty , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] It would be nice adding more sections to your readme file eg getting started section, also your MIT license needs to be personalized

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

d4nQw3rty committed:
update index

*Ensure logo is wrapped in an a tag.

d4nQw3rty committed:
update index

* Ensure the hamburger menu is wrapped in a nav tag.
*Change the headline <div> tag container  to a headline <section> tag container.
*Change <h2> tag to <h1> tag

d4nQw3rty committed:
Update README.md

*add getting started to readme file.

prolajumokeoni commented on README.md:
```diff
@@ -14,6 +14,12 @@ build the first 2 sections of the mobile website using the template you chose in
 - git and gitHub
 - Visual Code
 
```
### OPTIONAL

-  Well done making your readme file descriptive, kindly ensure all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) are present 👍🏿

prolajumokeoni commented on index.html:
```diff
@@ -5,9 +5,40 @@
+         <nav id="Logo-Placeholder">
+             <a href="#" id="My-Logo"><img src="img/My_logo.png" alt="Home page"></a>
+         </nav>
+         <div id="Normal-Button-Tertiary-Enabled">         
+            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
+         </div>
```
`<nav id="Normal-Button-Tertiary-Enabled">         
            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
         </nav>`

- [ ] Just as the previous reviewer requested, wrap the hamburger menu icon in a nav tag, currently you wrapped the logo in the nav tag and not the menu icon, please change this 👍🏿

prolajumokeoni reviewed (changes requested):
Hi @d4nQw3rty, 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

d4nQw3rty committed:
update index.html update README FILE

*Wrap the hamburger menu icon in a nav tag.
*Add Main headers to License and live demo

d4nQw3rty committed:
Update index.html

*open div fixed!

OmarRamoun reviewed (approved):
# 🟢 Project Approved! 🏆

## Hi @d4nQw3rty 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@omarramoun](https://img.shields.io/github/followers/omarramoun?label=Ramoun&style=social)](https://github.com/omarramoun)
- This is My **Twitter**: [![@omarramoun](https://img.shields.io/twitter/follow/omarramoun?label=ramoun16&style=social)](https://twitter.com/omarramoun)
- This is My **Slack**: [![Ramoun](https://img.shields.io/badge/slack-@Ramoun-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
